From 5253d444aebc5e414adb6af8a9a3fb08938cd4cc Mon Sep 17 00:00:00 2001
From: Denis Jagoudel <denis.jagoudel@ariadnext.com>
Date: Fri, 2 Dec 2016 15:59:04 +0100
Subject: [PATCH] Update tox.ini

---
 tox.ini | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tox.ini b/tox.ini
index fe94589..4c81bdb 100644
--- a/tox.ini
+++ b/tox.ini
@@ -4,7 +4,7 @@ envlist = py27, py35
 [testenv]
 deps=-r{toxinidir}/requirements.txt
      -r{toxinidir}/test-requirements.txt
-       
+
 commands=
    nosetests \
-      []
\ No newline at end of file
+      []
-- 
2.9.3 (Apple Git-75)

